bin/geoquantize

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+var commander = require("commander"),
+    vm = require("vm"),
+    d3 = Object.assign({}, require("d3-geo"), require("../")),
+    read = require("./read"),
+    write = require("./write");
+
+commander
+    .version(require("../package.json").version)
+    .usage("[options] [file]")
+    .description("Transform GeoJSON, such as to project from spherical to planar coordinates.")
+    .option("-o, --out <file>", "output file name; defaults to “-” for stdout", "-")
+    .option("-p, --precision <value>", "number of output digits after the decimal point")
+    .option("-n, --newline-delimited", "use newline-delimited JSON")
+    .parse(process.argv);
+
+if (commander.args.length > 1) {
+  console.error();
+  console.error("  error: multiple input files");
+  console.error();
+  process.exit(1);
+} else if (commander.args.length === 0) {
+  commander.args.push("-");
+}
+
+var sandbox = {d3: d3, d: undefined, i: -1},
+    context = new vm.createContext(sandbox),
```

Comment:
Don’t think the *sandbox* or *context* are needed for geoquantize.

Reply:
true! I also removed _vm_.